app/views/companies/_company_members.html.haml

```diff
@@ -0,0 +1,17 @@
+- applications = company.accepted_applications_from_members
+
+- if applications.any?
+
+  .col-md-12.company_member
```

Comment:
Table headers aren't needed if we make the business categories visually distinct from the names:

<img width="654" alt="co-member-employees-ul--narrow-viewport" src="https://user-images.githubusercontent.com/673794/33736611-1baf0c94-db48-11e7-98be-1bd51fb9d909.png">

in which case `ul` is the right element to use.  (I have the CSS and .erb for this if that's the way we want to go)

Reply:
Per you comment above:  Perhaps "Employees" isn't the right title/header to use.

Reply:
I like the table for now as 1) it better accommodates listing more than one or two employees, 2) ditto for an employee with _many_ categories, and 3) that real estate - for a company person or admin - is taken up by the payment status/button and the admin edit tools.

In any case, we can rearrange this later if desired.

Reply:
(just goes to show it's time for us to re-organize the layout, especially for company-members and admins)